src/ConsoleApp/consoleAppDemo.java

```diff
@@ -6,16 +6,90 @@
 
```

Comment:
First off, Ugly indentations is hard to read - not just for me, but for you. Use Ctrl + Shift + F to auto-format in eclipse.